homeassistant/components/directv/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for the DirecTV integration."""
```

Comment:
Please don't change unrelated things in this PR. Each PR should only accomplish one thing. This PR should only implement unique_id for this integration entities.

A separate PR is welcome where constants are clean up if needed.